src/telnetio/_machine.py

```diff
@@ -26,245 +22,263 @@ class TelnetOption:
     reply_pending: bool = False
 
 
+class State(Enum):
+    #: Receiving normal data
+    data = auto()
+    #: Receiving data after CR received
+    data_cr = auto()
+    #: IAC
+    command = auto()
+    #: IAC with multiple (3) bytes
+    negotiation = auto()
+    #: Subnegotation option
+    subn_option = auto()
+    #: Subnegotation sub-opion
+    subn_suboption = auto()
+    #: Subnegotation data
+    subn_data = auto()
+    #: Subnegotation end
+    subn_end = auto()
```

Comment:
This is a bit nit-picky, but I prefer to see enums as uppercase. We're already following this convention for `ErrorKind`. This follows PEP-8 re constants: https://peps.python.org/pep-0008/#constants and follows Guido's recommendation: https://mail.python.org/pipermail/python-ideas/2016-September/042340.html

Reply:
OK i can do that

Reply:
fixed